package.json

```diff
@@ -9,26 +9,27 @@
     "url": "https://github.com/juju/juju-gui.git"
   },
   "dependencies": {
+    "chai": ">=1.2.0",
     "cryptojs": ">= 2.5.3",
     "d3": "3.2.7",
-    "yui": "3.11.0",
-    "mocha": "1.8.x",
-    "express": "3.x",
     "expect.js": "0.1.2",
-    "should": ">=1.0.0",
-    "chai": ">=1.2.0",
-    "less": "1.3.x",
+    "express": "3.x",
+    "graceful-fs": "1.1.x",
+    "grunt": "0.3.17",
     "jshint": "2.1.9",
-    "recess": "1.1.x",
-    "node-markdown": "0.1.x",
-    "yuidocjs": "0.3.x",
+    "less": "1.3.x",
     "minimatch": "0.2.x",
-    "graceful-fs": "1.1.x",
+    "mocha": "1.8.x",
+    "node-markdown": "0.1.x",
+    "node-minify": "0.7.x",
+    "node-sass": "^3.1.2",
```

Comment:
This is the only addition, the rest was just the script making everything alphabetical.